src/v2/view-builder/views/email/BaseAuthenticatorEmailView.js

```diff
@@ -78,8 +78,15 @@ const Body = BaseFormWithPolling.extend(Object.assign(
       this.stopPolling();
 
       // Wait 1 min before polling again & do not show error message when encounter rate limit error
-      if (error.responseJSON?.errorSummaryKeys?.includes('tooManyRequests') ||
-        error.responseJSON?.errorCode === 'E0000047') {
+      if (
+        !error.responseJSON?.errorIntent &&
+        (error.responseJSON?.errorSummaryKeys?.includes('tooManyRequests')
+          || error.responseJSON?.errorCode === 'E0000047')
+      ) {
+        // if there is intent, ignore
+        if (error.errorIntent) {
```

Comment:
I understand what `!error.responseJSON?.errorIntent` (line 82) is, but what is this `error.errorIntent`?

Once I understand that, perhaps I will understand how, given that (IIUC) line 82 will only allow us to get here if `error.responseJSON.errorIntent` doesn't exist (or is otherwise falsy), this check here is behaving.

Reply:
I refactored the whole method so it should be more readable.

Reply:
Good, but does `error` _have_ an `errorIntent` property?  Or does it only have a `responseJSON?.errorIntent` property?

Reply:
you're correct. fixed it. ~stared at it way too long to want to admit before i spot it~

```
  let errors = {
    errorCauses: getRemediationErrors(response),
    errorSummary: getGlobalErrors(response),
    errorSummaryKeys: getGlobalErrorKeys(response),
    errorIntent: response.intent,
  };

  return {
    responseJSON: errors
  };
```